sdk/test-utils/perf/src/perfTest.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AbortSignalLike } from "@azure/abort-controller";
+import { BatchPerfTest } from "./batchPerfTest";
+
+/**
+ * Extends BatchPerfTest.
```

Comment:
Nit: This line doesn’t seem particularly useful to me.

Reply:
```suggestion
```